### PR TITLE
DEV-15150 remove unused variables

### PR DESCRIPTION
**Pull request by Tamr-Frank**
https://datatamr.atlassian.net/browse/DEV-15150

### CODE REVIEW

File: examples/minimal/main.tf [hunk 1]
```diff
@@ -8,7 +8,6 @@ module "rds_postgres" {
   username             = "exampleUsername"
   password             = "examplePassword" #tfsec:ignore:GEN003
 
-  vpc_id            = var.vpc_id
```
Comment (souza-dan) [inline examples/minimal/main.tf:11]: is var.vpc_id also now unused in this example and could be removed from examples/minimal/variables.tf?

**Review by souza-dan — COMMENTED** [review]
Looks good!

Could you run `make terraform/docs` so that the README gets updated?

File: examples/minimal/main.tf (hunk 1 above)
Comment (Tamr-Frank) [reply examples/minimal/main.tf:11]: It's used in the rds-postgres-sg in the example and I think it's a required field

**Review by souza-dan — APPROVED** [review]
LGTM

**Pull request merged**